### DISCUSSION TITLE
Attachments not synchronized on mobile device

### DISCUSSION THREAD

**Discussion by alpinfra** [opengisch/QField#2460] [Q&A]
Hi there, 

I have developed an input mask for QField mobile device, where the field team can view locally stored JPGs through the attachment widget. I am able to upload them to QFieldCloud, but I cannot view/access them on my mobile device running QField Dev. 

It works with regular QField.

I tried to find the locally stored QFieldCloud Project and move the JPG files in the local folder, but cannot find one. Mind that these JPGs are not intended to be edited or synchronized, I just need them copied once to the mobile device and be able to access them from the QField widget.
Also, when taking a picture in the field with the mobile device, synchronization works beautifully!

Any help?

Here is a snippet from the data list from this project, taken from a web browser. The JPGs were clearly uploaded:
![image](https://user-images.githubusercontent.com/57482615/141422498-8888cf8c-1327-4326-90ef-ae557134c890.png)

**Comment by marioba** [comment]
In order to provide a clearer and more manageable discussion forum and to have all discussions about QField and related products (i.e. QFieldCloud and QFieldSync) in a single place, we transferred all the discussions from the QFieldCloud to the [QField repository](https://github.com/opengisch/QField/discussions).

**Comment by suricactus** [comment]
@alpinfra Hey, sorry for the delayed answer. Can you test with the latest beta and/or nighly builds? There was a small issue regarding photo syncrhonization that was fixed recently.

Also, please make sure the attachment form fields are configured to use "relative path" on QGIS.

**Reply by suricactus** [reply]
@alpinfra sorry I was not clear, I meant QField beta/dev version.

Please install the latest QField from here [sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-master-70e3769da009796c632badc35f2949d4717de23a-arm64-android.apk](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-master-70e3769da009796c632badc35f2949d4717de23a-arm64-android.apk)

**Reply by alpinfra** [reply]
Will give it a try. However, I deleted the original project from the cloud before trying creating it again from scratch with the latest QFieldSync plugin. Need to figure out how to get it up on the cloud again ;-)

**Reply by suricactus** [reply]
Ah, I see why you posted the QFieldSync issue. :) Let's see if we can get it sorted https://github.com/opengisch/qfieldsync/issues/404 fast enough.

**Reply by alpinfra** [reply]
I was able to upload the project from an earlier stage to QFieldCloud.
Installed QField on my mobile device (Samsung tablet) from your APK. Checked version, it says "master-dev", updated on 2022-03-07.

On mobile device, when trying to download the project, I get an error: 
`Downloading error. Packaging job finished unsuccessfully for XX. Job (XX) finished with a failed status.`

When I open QFieldCloud in a web browser, project is listed with status OK.

**Reply by suricactus** [reply]
Thanks, will have a look.

**Reply by suricactus** [reply]
The crash is due to mbtiles, as you jave noted. Back to your photos: there is a limitation that you need to store all your photos in a directory named `DCIM` , while your files are stored in a directory named `JPG` . When you rename the directory don't forget to update your data in the attribute table of the photos layer too.

**Reply by alpinfra** [reply]
Am i able to use subdirectories within `DCIM`? Would that work?

**Reply by suricactus** [reply]
yes, subdirs within DCIM are fine.

**Reply by alpinfra** [reply]
YESSS! I can confirm it works 👍  moved the `/JPG` folder into `/DCIM/JPG` and I am now able to view the attachments :)

**Reply by suricactus** [reply]
Wohoo! :balloon:  Happy QFieldClouding!

**Comment by suricactus** [comment]
There is a limitation that you need to store all your photos in a directory named `DCIM`, while the files in your project are stored in a directory named `JPG`. You can have multiple subdirectories in the `DCIM` directory. When you rename the directory don't forget to update your data in the attribute table of the photos layer too.
